### PR TITLE
Implement the `export to calendar` functionality

### PR DESCRIPTION
**Pull request by bp323**
screen shot 2015-01-15 at 13 58 21

### CODE REVIEW

**Comment by bp323** [comment]
Reassigning to @Coenego for review.

File: tests/qunit/tests/js/api.util.js [hunk 1]
```diff
@@ -534,10 +534,11 @@ require(['gh.core', 'gh.api.tests'], function(gh, testAPI) {
     QUnit.test('renderTemplate - Partials', function(assert) {
         // Verify that a partial can be used to render a template
         // Add a template to the page
-        $('body').append('<script id="qunit-template-partial" type="text/template"><%= _.partial(\'calendar\', {\'data\': data}) %></script>');
+        $('body').append('<script id="qunit-template-partial" type="text/template"><%= _.partial(\'calendar\', {\'data\': data, \'gh\': gh}) %></script>');
         // Create the data to use in the template
         var data = {
-            'data': null
+            'data': null,
```
Comment (Coenego) [inline tests/qunit/tests/js/api.util.js]: Do we need to specify `data` if it's null?

File: apps/timetable/ui/js/index.js [hunk 2]
```diff
@@ -57,7 +57,8 @@ define(['gh.core', 'gh.subheader', 'gh.calendar', 'gh.student-listview', 'jquery
     var setUpCalendar = function() {
         // Render the calendar template
         gh.api.utilAPI.renderTemplate($('#gh-calendar-template'), {
-            'data': null
+            'data': null,
```
Comment (Coenego) [inline apps/timetable/ui/js/index.js]: Previous question also applies for this `data` object

**Comment by Coenego** [comment]
Left a question. Looks really good otherwise. Reassigning to @bp323

File: tests/qunit/tests/js/api.util.js (hunk 1 above)
Comment (bp323) [inline tests/qunit/tests/js/api.util.js]: I've removed it and everything works fine

**Comment by bp323** [comment]
Back to @Coenego for review.

**Pull request merged**